sovrin_client/agent/agent_verifier.py

```diff
@@ -53,3 +59,21 @@ def __init__(self, verifier: Verifier):
                              'claim issued by: {}'.format(", ".join(
                 sorted([sk.issuerId for sk in proof.schemaKeys]))))
             await self._postClaimVerif(claimName, link, frm)
+
+    def sendProofReq(self, link: Link, proofReqSchemaKey):
+        if self._proofRequestsSchema and (
+                    proofReqSchemaKey in self._proofRequestsSchema):
+            proofRequest = self._proofRequestsSchema[proofReqSchemaKey]
+            op = OrderedDict([
```

Comment:
Why an OrderedDict? To keep signatures same, if yes then `signAndSend` takes care of that